boards/px4/fmu-v4/ctrlalloc.cmake

```diff
@@ -91,6 +91,7 @@ px4_add_board(
 		sih
 		temperature_compensation
 		uuv_att_control
+		uuv_bluerov2_control
```

Comment:
Why is a controller just for bluerov2 required to be a separate module?

Is there a reason that you couldn't integrate this into the `uuv_att_control` module?